src/main/java/uk/gov/hmcts/reform/slc/logging/AppDependency.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.hmcts.reform.slc.logging;
+
+final class AppDependency {
+
+    /**
+     * Service bus message receiver dependency.
+     * Used to track dependency of how long in millis did it take to...
+     */
+    static final String SERVICE_BUS_DEPENDENCY = "ServiceBus";
+
+    /**
+     * .. receive message from service bus queue.
+     */
+    static final String SERVICE_BUS_MESSAGE_RECEIVED = "MessageReceived";
```

Comment:
how about we split this class into: `AppDependencies` and `AppDependencyCommands`?